config.example/group_vars/all.yml

```diff
@@ -96,3 +96,29 @@ users:
     groups: ['sudo']
     uid: 10007
     home: /home/nvidia
+
+# Zabbix
+# Web User: Admin, Pass: zabbix
+postgresql_users:
+  - name: zabbix-server
+    password: zabbix-server
+zabbix_url: 'http://{{ hostvars[groups["slurm-master"][0]]["ansible_fqdn"] }}:{{ zabbix_apache_vhost_port }}'
+zabbix_server_name: "{{ ansible_fqdn }}"
+zabbix_server_hostname: "{{ ansible_fqdn }}"
+zabbix_version: 4.4
+zabbix_apache_vhost_port: 8080
+zabbix_apache_vhost_tls_port: 8443
+zabbix_timezone: America/Los_Angeles
+zabbix_server_pgsql_login_host: localhost
+zabbix_server_pgsql_login_user: zabbix-server
+zabbix_server_pgsql_login_password: zabbix-server
```

Comment:
Why not use MySQL? Since it's already installed on the `slurm-master` node anyway. :)

Reply:
Using MySQL was actually a major disaster, it uninstalled MariaDB without prompting and installed MySQL, which broke Slurm. I agree it would be ideal to use a single DB, so I can keep trying.